### PR TITLE
Add BestEffort definition

### PR DESCRIPTION
**Pull request by brennv**
Wording from https://kubernetes.io/docs/admin/resourcequota/walkthrough/

### CODE REVIEW

File: admin_guide/quota.adoc [hunk 1]
```diff
@@ -149,6 +149,10 @@ A *BestEffort* scope restricts a quota to limit the following resources:
 
 - `*pods*`
 
+*BestEffort* scoped pods can take advantage of slack compute resources in their
+namespace, and then require that users make an explicit resource request for
+pods that require a higher quality of service.
```
Comment (bparees) [inline admin_guide/quota.adoc]: this still doesn't really tell me how i create a besteffort pod.  

The other page says that the BestEffort scope will "Match pods that have best effort quality of service for either cpu or memory."

I still don't understand how I set memory or cpu to "best effort"  (the reality, I believe, is that if you set a limit, but no request, for a given resource(eg memory or cpu) that is considered best effort for that resource, but I don't think we state that explicitly anywhere)

Comment (brennv) [inline admin_guide/quota.adoc]: Good point, maybe we need a new section with examples drawing inspiration from https://github.com/kubernetes/community/blob/master/contributors/design-proposals/resource-qos.md#qos-classes

Comment (derekwaynecarr) [inline admin_guide/quota.adoc]: A BestEffort pod can use slack compute resources that are unused on its scheduled node.

A BestEffort pod is a pod that does not have a compute resource requirement for cpu and memory.

**Review by derekwaynecarr — CHANGES_REQUESTED** [review]

File: admin_guide/quota.adoc (hunk 1 above)
Comment (derekwaynecarr) [inline admin_guide/quota.adoc]: or just link to https://docs.openshift.org/latest/admin_guide/overcommit.html#qos-classes ?

**Comment by ahardin-rh** [comment]
This is an old one, but is it about ready to go? Once this is updated, can you please give it a squash so that I can merge? Thanks!

**Comment by ahardin-rh** [comment]
Continued work in https://github.com/openshift/openshift-docs/pull/5012

**Pull request closed without merging**